WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+- **Range:** Self
+- **Components:** V, S
+- **Duration:**  Concentration, up to 1 minute
+___
+Ice rapidly materializes around you, protecting you from harm. You gain 40 temporary hit points and immunity to cold damage for the spells duration. While the spell is active you can't move and take any actions or reactions. The spell remains until the temporary hit points have been expend or until your concentration breaks.
+
+#### Ice Nova
+*8th-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 120 feet
+- **Components:** V, S, M (glass dust and water)
+- **Duration:** Instantaneous
+___
+Frigid cold rapidly expands in a 15-foot-radius centered on a point you you can see within range. Each creature in the cold must succeed on a Dexterity saving throw. On a failed save, a creature takes 10d6 cold damage and is restrained for 1 minute as ice encases the lower part of their legs. On a successful save, it takes half as much damage and isn't restrained by this spell.
+
+A creature restrained by the ice can use its action to make a Strength check against your spell save DC. On a success, it frees itself. Alternatively can the ice be broken by dealing a total of 40 hit points worth of damage to it, its armor class is 10. 
+
+#### Jaina's Flying Ship
+*9th-level transmutation*
+___
+- **Casting Time:** 1 minute
+- **Range:** Touch
+- **Components:** V, S
+- **Duration:** 24 hours
+___
+You channel your powers into a nonmagical ship within range, the ship can be no larger than 100 feet long, and 30 feet wide. For the spells duration, the ship gains a flying speed of 60 feet, the ship changes direction and altitude at your command, continuing its course until a new command is given or the spells duration ends.
+
+The flying ship has an Armor Class of 15, and 300 hit points; immunity to poison and psychic damage, and immunity to all conditions.
+
+<div class='footnote'>PART 2 | SPELLS </div>
+
+\pagebreakNum
+
+It has a maximum carrying capacity of 50 tons (limited by the size of the ship itself). It is not slowed by any weight until its maximum capacity is reached, at which point the ship loses its flying property and falls at a rate of 60 feet per round until it hits the ground. Taking no falling damage from the fall, and leaving its passengers unharmed.
+
+You can this effect permanent by casting this spell on the same ship every day for one year.
+
+#### Living Bomb
+*2nd-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 60 feet
+- **Components:** V, S, M (a flammable liquid)
+- **Duration:**  Concentration, up to 1 minute
+___
+You manipulate the mana flowing through a creature within range, making it burn from the inside with a roaring fire. The target must make a Constitution saving throw. Taking 1d10 fire damage on a failed save, or half as much on a successful one. At the start of each of your turns for the spells duration, the target must make another Constitution saving throw as it takes the damage again.
```

Comment:
Should there be a full . stop before "taking 1d10 fire damage"? 

Also "the spell's duration" here as well.

Reply:
Better?

The target must make a Constitution saving throw. On a failed save it takes 1d10 fire damage, or half as much on a successful one.

Reply:
That looks pretty good. 👍